cluster/helm/splice-validator/templates/validator.yaml

```
@@ -129,9 +129,17 @@ spec:
             canton.validator-apps.validator_backend.scan-client.seed-urls = [ ${_scan.admin-api.address} ]
             {{- end }}
         {{ end }}
-        {{ if not .Values.useSequencerConnectionsFromScan }}
+        {{ if not .Values.useSequencerConnectionsFromScan -}}
         - name: ADDITIONAL_CONFIG_STATIC_SEQUENCER_URL
           value: canton.validator-apps.validator_backend.domains.global.url = {{ .Values.decentralizedSynchronizerUrl | quote }}
+        {{ else if .Values.synchronizer -}}
+            {{- if eq .Values.synchronizer.synchronizerType "trusted-url" }}
```

IIRC I was asked to bikeshed somewhere here.

So here it goes...

```suggestion
            {{- if eq .Values.synchronizer.connectionType "trusted-url" }}
```

(and all the other instances accordingly, if you accept that)

---

Resolved: Applied